manifests/s/Stardock/Fences/4.11.5/Stardock.Fences.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Stardock.Fences
+PackageVersion: 4.11.5
```

```suggestion
PackageVersion: 4.1.1.5
```
This must match Control Panel.

You also have to change `manifests/s/Stardock/Fences/4.11.5` to `manifests/s/Stardock/Fences/4.1.1.5`

---

Updated to 4.1.1.5 for path and PackageVersion